tests/test_group.py

```diff
@@ -102,12 +102,12 @@ def test_group_commands(user_factory, invoke_as_superuser):
     )
     assert res.retcode == 0
     assert res.data["id"] == group2_id
-    assert res.data["user_ids"] == [user2_id, user3_id]  # order
+    assert res.data["user_ids"] == [user3_id, user2_id]  # order
```

Comment:
Order is undefined, unless the endpoint enforces it in the sql query. In tests, I'd rather use sets.